test/gov/hzn_sdo.sh

```diff
@@ -67,6 +67,30 @@ cat <<'EOF' > /tmp/sdo_key2.json
 }
 EOF
 
+cat <<'EOF' > /tmp/sdo_key_unsupported.json
```

Comment:
Why is this metadata unsupported and why don't i see it used anywhere?

Reply:
Because I used the wrong name in the test. Thanks for the catch.